host-interaction/bootloader/disable-code-signing.yml

```diff
@@ -0,0 +1,14 @@
+rule:
+  meta:
+    name: disable code signing
+    namespace: host-interaction/bootloader
+    author: william.ballenthin@fireeye.com
+    scope: function
+    att&ck:
+      - Defense Evasion::Subvert Trust Controls::Code Signing Policy Modification [T1553.006]
+    examples:
+      - 0596C4EA5AA8DEF47F22C85D75AACA95:0x10710B3  # old Necurs rootkit
+  features:
+    - and:
+      - match: create process
+      - string: /^bcdedit(\.exe)? -set TESTSIGNING ON/i
```

Comment:
this is more commonly seen in BAT scripts (possibly dropped by malware) but, even then, there aren't too many examples of this technique within a program